src/components/RegistrationForm.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import RegistrationForm from "./RegistrationForm";
+
+jest.mock("../store/actionCreators");
+jest.mock("../components/fetchWrapper");
+
+const testValues = {
+  name: "User",
+  email: "user@user.com",
+  password: "123456",
+};
+
+const setup = () => {
+  const utils = render(<RegistrationForm />, { wrapper: BrowserRouter });
+  const nameField: HTMLInputElement = screen.getByLabelText("name");
+  const emailField: HTMLInputElement = screen.getByLabelText("email");
+  const passwordField: HTMLInputElement = screen.getByLabelText("password");
+  const repeatField: HTMLInputElement = screen.getByLabelText("repeatpassword");
+  return {
+    nameField,
+    emailField,
+    passwordField,
+    repeatField,
+    ...utils,
+  };
+};
+
+describe("Login form test:", () => {
+  it("should render the component", () => {
+    render(<RegistrationForm />, { wrapper: BrowserRouter });
+    expect(screen.getByTestId("form")).toBeInTheDocument();
+  });
+
+  it("should get the url with the path= /login if user click on link", () => {
+    render(<RegistrationForm />, { wrapper: BrowserRouter });
+    fireEvent.click(screen.getByText("Авторизоваться"));
+    expect(global.window.location.href).toContain("http://localhost/login");
+  });
+
+  it("should check fields of form when user tries to fill them in", () => {
```

Comment:
I'm not sure that the sense of the statement is relevant for the asserts below it